src/main/java/murphy/task/Deadline.java

```diff
@@ -0,0 +1,47 @@
+package murphy.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import murphy.MurphyException;
+
+public class Deadline extends Task {
+    protected LocalDate by;
```

Comment:
`by` should have a more proper name, such as `ddlDate`.

Reply:
the name `by` would be sufficient to tell the context.